src/index.html

```diff
@@ -1,12 +1,492 @@
 <!doctype html>
-<html lang="en">
+<html lang="en" class="page">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
-  <body>
-    <h1>Catalog</h1>
+
+  <body class="page__body">
+    <header class="header page__header">
+      <a href="#home" class="header__logo-link">
+        <img
+         src="/images/logo.png"
+         alt="logo"
+         class="header__logo-img"
+        >
+      </a>
+
+      <nav class="nav header__nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a
+             href="#apple"
+             class="nav__link is-active"
+            >
+             Apple
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+             href="#samsung"
+             class="nav__link"
+            >
+             Samsung
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+             href="#smartphones"
+             class="nav__link"
+            >
+             Smartphones
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+             href="#laptops%20&%20computers"
+             data-qa="nav-hover"
+             class="nav__link"
+            >
+             Laptops & Computers
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+             href="#gadgets"
+             class="nav__link"
+            >
+             Gadgets
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+             href="#tablets"
+             class="nav__link"
+            >
+             Tablets
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+             href="#photo"
+             class="nav__link"
+            >
+             Photo
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+             href="#video"
+             class="nav__link"
+            >
+             Video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="page__main">
+      <div class="card page__card" data-qa="card">
+        <img
+          class="card__image"
+          src="./images/imac.jpeg"
```

Comment:
Use a relative path for the image source. Change the src attribute value from './images/imac.jpeg' to './images/imac.jpeg'